src/frontend/app/shared/components/list/list-types/cf-users/cf-permission-cell.ts

```diff
@@ -54,9 +59,17 @@ export abstract class CfPermissionCell<T> extends TableCellCustom<APIResource<Cf
         const permission = chip.key;
         this.removePermissionWarn(permission);
       };
-      chipConfig.hideClearButton$ = this.canRemovePermission(perm.cfGuid, perm.orgGuid, perm.spaceGuid).pipe(
-        map(can => !can),
-      );
+      if (perm.string === UserRoleLabels.org.short.users) {
+        // If there are other roles than Org User, disable clear button
+        chipConfig.hideClearButton$ = this.userEntity.pipe(
+         filter(p => !!p),
+         map( (entity: CfUser) => this.cfUserService.hasRoles(entity))
+        );
+      } else {
+        chipConfig.hideClearButton$ = this.canRemovePermission(perm.cfGuid, perm.orgGuid, perm.spaceGuid).pipe(
```

Comment:
The permissions check still need to be applied. ATM a non-manager can remove an org user role if it's the only role that user has